sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/util/Beta.java

```diff
@@ -37,7 +37,7 @@
     /**
      * Azure library version numbers
      */
-    enum SinceVersion {
+    public enum SinceVersion {
```

Comment:
Is this intentional ?

Reply:
Yes, this was removed earlier in one of the PRs, so reverted it back.